CONTRIBUTING.md

```diff
@@ -66,11 +66,19 @@ compromise among collaborators be the default resolution mechanism.
 
 # Becoming a Member and Collaborator
 
-To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, investigate issues tagged as `good first issue`,
+To become a Member of the Community Committee, start participating! Attend scheduled meetings, investigate issues tagged as `good first issue`,
 file issues and pull requests, and provide insight via GitHub. Request to become an Observer by
 filing an issue. Once added as an Observer to meetings, we will track attendance and participation
-for 3 months, as according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee). If you meet the minimum
-attendance and are participating, the CommComm will vote to add you as a member.
+according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee).
+Once you meet the minimum attendance and participation requirements of a Member, you will automatically become a Member.
+To help expedite the process, you are encouraged to open a PR adding yourself, but it is not a requirement. The Chair is
+responsible for keeping the Member list up-to-date, according to the rules outlined here (a PR with approvals is _not_
+required for this).
+
+If any CommComm member has and objection to an Observer eventually becoming a Member, they must raise that objection to
+the Chair, in private, prior to the Observer's eligibility. Resolution steps should be pursued in favor of removing the
```

Comment:
I am -1 on this:

> prior to the Observer's eligibility

This creates a timeboxed environment where all CommComm members will need to keep track of all other CommComm members to voice concerns. This isn't really conducive to an open/welcoming environment for our members. Time is already our most precious resource (and will be for the foreseeable future), and this drains that resource even more.

Reply:
How would a chair would responsibly track this? It can be difficult to maintain a level of trust with the members of CommComm by just saying 'someone came to me in private with a problem'. I know we've seen this happen in real world with various leaders and I think it puts the specific leader in a pickle when all they can say is that(I don't want them to be set up to not be trusted). Is there a precedence we could find somewhere(potentially some other project's governance) that handles private reportings in a reasonable way that doesn't erode trust?

Reply:
I do agree with @hackygolucky and @bnb , having this kind of matter to be brought privately in such a closed timebox can be both real hard to manage, and may lead to nefarious atmosphere IMHO. Total openness may not be perfect, but at least objections can be dealt with in a responsible and adult way, in accordance with the CoC. Reporting privately to the chair(s) makes me feel a bit like going to the teacher tbh, and I fear it may lead to general mistrust and drive people away.

Reply:
@bnb, A CommComm member would have 3 months to raise an objection (Min attendance requirement). So, yeah, this is timeboxed- but it's a pretty big box. Sure- if a member jumps in at the last moment with an objection- it could cause problems. Maybe your first question to the person Objecting would be "why are you objecting now and not 3 months ago?"

@hackygolucky, Someone should be in charge of keeping track of member's concerns. Given this past year's complaints- we need to take this seriously and being proactive about it is preferred. I believe this should be the Chair(s). This is not intended to be "I didn't like how they objected to my PR." Something like that the Chair should reply "make a comment on Github!" Instead, this is for concerns like "Hey this person has tweets that marginalize [a group] and we should open a dialog about it." In a scenario like that, we should have a conversation with the person and explain how communications (tweets) like that have reflected poorly on the project and the other Members.

The process needs to provide a safe and encouraging path for Members to raise a concern. However, just because the process outlines that it is done in private, doesn't mean they member reporting it needs to stay annon- it's their choice.

@Tiriel, this is not tattling- it is evaluating during the existing evaluation period. The existing process allows a person to do everything that is written and then a private vote happens. If that vote yields a "we don't accept you as a member" then the person and the Members are completely in the dark about why and there's no path forward. (Would we then drop them as an Observer? I don't know)

Reply:
> Maybe your first question to the person Objecting would be "why are you objecting now and not 3 months ago?"

@williamkapke How is any individual member of the CommComm supposed to know a member when they first attend a meeting? While you and I both know many people involved in Node.js, we're trying to make the CommComm an open and welcoming environment for literally _every_ user of Node.js that may have an interest in contributing. By suggesting that the default is that we object as soon as someone requests to be an observer, you're assuming the CommComm already knows the individual. Outside of the issue around shutting down people who are interested in contributing, this kind of approach is perpetuating the insider mentality that the project is perceived to have from those who aren't deeply involved, which is something we need to be deconstructing not strengthening and codifying.

> Given this past year's complaints [...]